### PR TITLE
html-form

### PR DESCRIPTION
**Pull request by pr1syazhniy**
- [[DEMO LINK](https://pr1syazhniy.github.io/layout_html-form/)]
- [[TEST REPORT LINK](https://pr1syazhniy.github.io/layout_html-form/)]

### CODE REVIEW

**Review by slavanosatskyi — CHANGES_REQUESTED** [review]
Please, add demo and report links to your PR description
And please correct your html indentations according to checklist
Screenshot 2023-02-28 at 18 43 29

**Comment by pr1syazhniy** [comment]
- [DEMO LINK](https://<pr1syazhniy>.github.io/layout_html-form/)
- [TEST REPORT LINK](https://<pr1syazhniy>.github.io/layout_html-form/report/html_report/)

Не зовсім розумію як правильно відкоректувати код так,щоб пройшла команда npm run lint.
Зараз проходить,але як тільки починаю форматувати як в інструкції [Code style] Good example,купа помилок вилізає.
Також через команду alt+shift+f в vs code автоматичне виправлення,усе з'їжджає так як у bad expample.

**Review by slavanosatskyi — CHANGES_REQUESTED** [review]
Please, correct your DEMO and REPORT links. It should look like this https://pr1syazhniy.github.io/layout_html-form/

Try to edit your code according to checklist and good examples. If there are any issues feel free to contact us in corresponding fe_chat

**Comment by pr1syazhniy** [comment]
Все виправив.

File: src/style.css [hunk 1]
```diff
@@ -1 +1,11 @@
-/* styles go here */
+.form-field {
+  display: block ;
+}
+
+.element {
+  margin-bottom: 10px;
+}
```
Comment (nazarmatsevych) [inline src/style.css]: You dont have element class

File: src/index.html [hunk 2]
```diff
@@ -2,16 +2,167 @@
...
+          Personal information
+        </legend>
+        <label class="form-field">
+          Surname:
+          <input 
+          type="text"
+          name="Surname" 
+          autocomplete="off" 
+          required
+          >
```
Comment (nazarmatsevych) [inline src/index.html]:
```suggestion
          <input 
            type="text"
            name="Surname" 
            autocomplete="off" 
            required
          >
```

image

Fix everywhere

File: src/index.html [hunk 3]
```diff
@@ -2,16 +2,167 @@
...
+        </label>
+      </fieldset>
+      <fieldset class="form-box">
+        <legend>
+            An interesting fact about you!
+        </legend>
+        <div class="form-field">
+            Do you love cats?
```
Comment (nazarmatsevych) [inline src/index.html]:
```suggestion
        <div class="form-field">
           Do you love cats?
```

image

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Check comments and proofread your code with [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

Also add correct margin to fix failed tests

File: src/style.css [hunk 4]
```diff
@@ -1 +1,8 @@
-/* styles go here */
+.form-field {
+  display: block ;
+  margin-bottom: 10px;
+}
+
+.form-fieldset {
+  margin-bottom: 20px;
+}
```
Comment (nazarmatsevych) [inline src/style.css]: You don't have this class in your html code.

Also don't forget to re-deploy your changes.

If you have any questions - feel free to ask them in fe_chat

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]

**Review by slavanosatskyi — APPROVED** [review]
Accepted, but next time do not forget to add test report, it is a mandatory requirement

**Pull request closed without merging**